lua/autorun/client/cl_contact_forms.lua

```diff
@@ -810,4 +810,42 @@ net.Receive( "CFC_ContactForms_FailureAlert", function()
     )
 end )
 
+net.Receive("CFC_ContactForms_Alert", function()
+    local data = net.ReadTable()
+    local reporter = player.GetBySteamID(data["steam_id"])
+    local reported = player.GetBySteamID(data["reported_steam_id"])
+
+    local reporterName = reporter:GetName()
+    local reporterRankColor = team.GetColor(reporter:Team())
+
+    local reportedName = reported:GetName()
+    local reportedRankColor = team.GetColor(reported:Team())
```

Comment:
Style guidelines, always make sure entities are valid
```suggestion
    local reporter = player.GetBySteamID( data.steam_id )
    local reported = player.GetBySteamID( data.reported_steam_id )

    if not IsValid( reporter ) or not IsValid( reported ) then return end

    local reporterName = reporter:GetName()
    local reporterRankColor = team.GetColor( reporter:Team() )

    local reportedName = reported:GetName()
    local reportedRankColor = team.GetColor( reported:Team() )
```

Reply:
👍